source/Login/WeeklyOverview/WeeklyStyles.css

```diff
@@ -217,7 +217,7 @@ span {
 /* css for settings */
 #settings {
     background-color: #ffffff;
-    width: 242px;
+    width: 280px;
```

Comment:
Not sure if we should be hard coding these values, especially the widths of elements

Reply:
discussed in meeting changes, leaving for now